ci/code_checks.sh

```diff
@@ -80,6 +80,13 @@ if [[ -z "$CHECK" || "$CHECK" == "lint" ]]; then
     flake8-rst doc/source --filename=*.rst --format="$FLAKE8_FORMAT"
     RET=$(($RET + $?)) ; echo $MSG "DONE"
 
+    # Check if pandas is referenced as pandas, not *pandas* or Pandas
+    MSG='Checking if pandas reference is standardized or not' ; echo  $MSG
+    grep -nr '*pandas*|Pandas' doc/*
+    grep -nr '*pandas*|Pandas' web/*
```

Comment:
I guess the next code works. You can't have two statements, since you are checking the exit code later, and you're only checking the code of the last statement.
```suggestion
    grep -nr '*pandas*|Pandas' web/* doc/*
```
Also, please use `invgrep` as in the rest of this script, you're doing here the opposite of what you want to do. And we have lots of validations where we already do this for other patterns, don't need to reinvent the wheel, just so the same, so we also keep consistent.